CHANGELOG.asciidoc

```diff
@@ -79,6 +79,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add docker diskio stats on Windows. {issue}6815[6815] {pull}8126[8126]
 - Fix incorrect type conversion of average response time in Haproxy dashboards {pull}8404[8404]
 - Fix dropwizard module parsing of metric names. {issue}8365[8365] {pull}6385[8385]
+- Add container image for docker metricsets. {issue}8214[8214] {pull}8438[8438]
```

Comment:
Could you please move it to the added section? This is for bugfixes I beleive

Reply:
fixed